src/modules/esl-share/core/esl-share-list.ts

```diff
@@ -12,8 +13,10 @@ import type {ESLShareButtonConfig} from './esl-share-config';
  *
  * ESLShareList is a custom element to dynamically draw {@link ESLShareButton}s using simplified shared config
  */
+@ExportNs('ShareList')
 export class ESLShareList extends ESLBaseElement {
   public static override is = 'esl-share-list';
+  public static observedAttributes = ['list'];
```

Comment:
```suggestion
  public static override observedAttributes = ['list'];
```

Reply:
This member cannot have an `override` modifier because it is not declared in the base class `ESLBaseElement`.